utils/binpath.go

```diff
@@ -51,6 +56,18 @@ func GetBinaryPath(command string) (string, error) {
 		}
 	}
 
+	// look for the binaries in brew symlink directories
+	// example: /usr/local/opt/e2fsprogs/bin, /usr/local/opt/mtools/bin etc.
+	for _, p = range BrewSpecificPaths {
+		items, err := os.ReadDir(p)
```

Comment:
We need to skip files in case there are some.

Reply:
I added the check to skip all non-directory items.